### ISSUE TITLE
(Enhancement) Colored Menu Options as in OFW Menus

### ISSUE THREAD

**Issue by Eldurinn** [400plus/400plus#392]
Just a minor little qualm for me is that 400Plus's menus are all red/gray, but would it be possible to customize them to the blue and yellow ones like OFW menus have? (either automatically or enabled via a setting)

**Comment by eduperez** [comment]
If I remember correctly, it can be done: there is a list of available colors at the beginning of "menu.h", and we just have to initialize the correct palette with "GUI_PaletteChange" at "menu.c".

**Comment by Eldurinn** [comment]
Oh, cool. I guess there's no easy way to invoke a menu option for it?  Or compile versions with the other color schemes.

**Comment by eduperez** [comment]
No, there is no menu option to change those colors, and it is not immediate to create a new one. Changing the code to another fixed value and compiling a new version is much easier; but right now I am busy with other issues that seem more important.

**Comment by Eldurinn** [comment]
Okay. That's a good point.  Keep up the great work. Been really enjoying using 400Plus!
